src/main/java/org/opentripplanner/api/mapping/I18NStringMapper.java

```diff
@@ -5,6 +5,16 @@
 
 public class I18NStringMapper {
 
+  private Locale locale;
+
+  I18NStringMapper(Locale locale) {
+    this.locale = locale;
+  }
+
+  public String mapToApi(I18NString string) {
+    return string == null ? null : string.toString(locale);
+  }
+
   static String mapToApi(I18NString string, Locale locale) {
```

Comment:
Can we remove this?

Reply:
There is couple of instances that uses this static method with `null` locale. Because of that I decided to leave this method in place, but I can remove this as well if you want.